GOVERNANCE.md

```diff
@@ -0,0 +1,82 @@
+# CoreDNS Governance
+
+## Principles
+
+The CoreDNS community adheres to the following principles:
+
+- Open: CoreDNS is open source.
+- Welcoming and respectful: See [Code of
+  Conduct](https://github.com/coredns/coredns/CODE-OF-CONDUCT.md).
+- Transparent and accessible: Work and collaboration are done in public.
+- Merit: Ideas and contributions are accepted according to their technical merit and alignment with
+  project objectives, scope, and design principles.
+
+## Benevolent Dictator for Life
+
```

Comment:
We cannot continue with that as a rule.

Reply:
Why actually? Not saying I would push for this? Is this a cncf thingy?

Reply:
The initial comment was my personal reaction ... 

About the CNCF position, officially it says that it requires only that the governance is stated and clarified. It does not require any particular kind of governance. However, all the projects at Incubator or Graduated level are using votes (considered as an 'open governance').
There were some talk in CNCF about the need to require open-governance (votes for decision) for all projects, or not. As I said above, the official position it to require only that the gobernance is defined.

However, in order to graduate, we need 2/3 of the votes of ToC, and I am thinking that some would refrain to vote if we show the 'dictator'-like governance (that is my personal opinion). Specifically we would be the first project with a non-open governance.

I raised the concern of such governance on last maintainer meeting were Infoblox were represented. I think that Alan/Cricket are reaching to you by email in order to deal the  "Corporate backing" and "Open-governance".
Please, take care to check those email and reply as we need to go ahead quickly on the Governance definition.
Thanks!